lib/explorer/polars_backend/series.ex

```diff
@@ -236,6 +236,13 @@ defmodule Explorer.PolarsBackend.Series do
   @impl true
   def any?(series), do: Shared.apply_series(series, :s_any)
 
+  @impl true
+  def row_index(series) do
+    Explorer.DataFrame.new(series: series)
+    |> Explorer.DataFrame.mutate_with(&[row_index: Series.row_index(&1[:series])])
+    |> Explorer.DataFrame.pull(:row_index)
```

Comment:
I wonder if we should expose this function in Rust instead? https://docs.rs/polars/latest/polars/datatypes/struct.UInt32Type.html#method.new 🤔 would it have any performance difference?

Reply:
And perhaps `new(lazy: true)`, but I'm not sure about allocations.

Thanks for the reference, I've pushed a commit exposing it in Rust.